charts/argo-rollouts/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v1.6.6
 description: A Helm chart for Argo Rollouts
 name: argo-rollouts
-version: 2.35.1
+version: 2.36.0
```

Comment:
nits: It's a backwards compatible optional feature, so bumping patch version is better. 🙋 
Ref: https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#new-application-versions

Reply:
Yeah works for me. Was just following the example of https://github.com/argoproj/argo-helm/pull/2590